### ISSUE TITLE
Not drawing a single box after training from scratch on ICDAR-2015

### ISSUE THREAD

**Issue by msds18004** [ZJULearning/pixel_link#134]
I trained my new model on ICDAR-2015 by training for 1st 100 steps with 0.001 learning rate and rest at 0.01 learning rate but after around 20000 iterations, it is not drawing even a single bbox on test images.

Also, the image size in train.sh is set to 512x512,  do I need to change image size in train.sh to 1280x720 as the ICDAR images are of this size, also their annotations are according to this size. Can this be the issue or is it already handled in image_resize code?

How do you train the 4s provided model (model.ckpt-38055)? It gives good evaluations on test images
Do I have to change something else?
I wanted to test it out as I am planning to fine tune it on my own dataset.

Kindly someone guide me how to fix it

**Comment by VadymBezdushnyi** [comment]
Got the same issue. The predicted mask is just zeros

**Comment by Pro-flynn** [comment]
Have you solved this issues?
if you have solved, could you give me some tips?
Thanks！

**Comment by ajinkya933** [comment]
@msds18004 @VadymBezdushnyi @Pro-xiaowen @comzyh @dengdan  any idea how to solve it

**Comment by pranshu19chester** [comment]
Hi, has anyone found a solution to this problem? I tried training on SROIE2019 dataset for 5,00,000 steps but my model didn't make any bboxes.

**Comment by mohamad-hoseini** [comment]
Training works on icdar2015 dataset but I have the same problem on my own dataset. Could you manage to solve it?
@pranshu19chester  @VadymBezdushnyi  @Pro-xiaowen @ajinkya933 @msds18004

**Comment by pranshu19chester** [comment]
@mohamad-hoseini try checking the config.py that you are using. There is a default config.py in the root folder. But a different config.py exists when you download the pre-trained model. For me, the root folder config.py file was the wrong one due to which my model wasn't being trained properly. I altered the config.py that came with the pre-trained model and used that.

**Comment by arshv27** [comment]
@pranshu19chester Have you tried training on a custom dataset? I'm trying to train on ICDAR2019 MLT, but I am facing several issues.  A lot of them have been documented but fixes haven't been suggested. Did you manage to complete your task?
